plugin.video.orftvthek/addon.xml

```diff
@@ -1,25 +1,33 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.orftvthek" name="ORF TVthek" version="0.10.3" provider-name="sofaking">
+<addon id="plugin.video.orftvthek" name="ORF TVthek" version="0.12.0" provider-name="sofaking">
     <requires>
         <import addon="xbmc.python" version="2.14.0"/>
-        <import addon="script.module.future" version="0.16.0"/>
-        <import addon="script.module.kodi-six" version="0.0.5"/>
-        <import addon="script.module.simplejson" version="2.0.10" optional="false"/>
+        <import addon="script.module.future" version="0.17.1"/>
+        <import addon="script.module.kodi-six" version="0.0.7"/>
+        <import addon="script.module.simplejson" version="3.16.1"/>
+        <import addon="script.module.inputstreamhelper" optional="true"/>
     </requires>
     <extension point="xbmc.python.pluginsource" library="default.py">
         <provides>video</provides>
     </extension>
     <extension point="xbmc.addon.metadata">
         <platform>all</platform>
         <language>de en</language>
-        <summary lang="de">ORF TVthek</summary>
-        <summary lang="en">ORF TVthek</summary>
-        <description lang="de">ORF TVthek - Ermöglicht Ihnen den Zugriff auf die ORF TVthek Video Platform</description>
-        <description lang="en">ORF TVthek - This plugin provides access to the Austrian "ORF TVthek"</description>
+        <summary lang="de_DE">ORF TVthek</summary>
```

Comment:
I don't think this is available in gotham. Why don't you consider to move this to leia or krypton? I don't believe you actually have runtime tested this under gotham :)
it's dead

Reply:
well not exactly since my ancient pi1 with the gotham install killed the sdcard :(. however i did run the kodi-addon-checker for gotham without any errors. i'm trying to support the old versions as long as possible, because people are still using gotham according to this http://mirrors.kodi.tv/stats?downloadstats=2021&filter=plugin.video.orftvthek